pandas/tests/plotting/test_datetimelike.py

```diff
@@ -1104,6 +1104,33 @@ def test_time(self):
                     xp = time(h, m, s).strftime('%H:%M')
                 assert xp == rs
 
+    @pytest.mark.slow
+    @pytest.mark.skip(reason="Unreliable test")
```

Comment:
you need to mark it so a xfail pass won’t faiil

Reply:
I would also add `strict=False` to this too given the flakiness.

Reply:
Yep thanks - didn't realize we set this to True in the config. Updated now!